salt/states/boto_dynamodb.py

```diff
@@ -162,6 +162,11 @@
 
 import salt.utils.dictupdate as dictupdate
 
+__deprecated__ = (
+    3009,
+    "boto",
+    "https://github.com/salt-extensions/saltext-boto",
+)
```

Comment:
I know it's out of scope, but please let's also take the time to remove the `logging.basicConfig` call that should have never been merged in.

Reply:
Can also be on a follow-up PR.

Reply:
Let's do it in a follow-up PR